docs/content/en/docs/Coding Milestones/PoC2023q1/scripts.md

```diff
@@ -61,28 +70,39 @@ spec:
 EOF
 ```
 
-This script operates in an idempotent style.  It looks at the current
-state and makes whatever changes are needed.  Caveat: it does not cast
-a skeptical eye on the spec of a pre-existing Location.
+This command operates in idempotent style, making whatever changes (if
+any) are needed to move from the current state to the desired state.
+Current limitation: it does not cast a skeptical eye on the spec of a
+pre-existing Location.
 
 ## Removing SyncTarget/Location pairs
 
 The following script undoes whatever remains from a corresponding
-usage of `ensure-location.sh`.  Invoke it with the inventory
-management workspace current.
+usage of `kubectl kcp-edge ensure location`.
+
+This commad accepts all the global command-line options of `kubectl`
+excepting `--context`.
+
+Invoke this command when your current workspace is your chosen
+inventory management workspace, or specify that workspace with the
+`--imw` command line flag.  Upon completion, the current workspace
+will be your chosen inventory management workspace.
+
+This command does not depend on the action of any of the edge-mc
+(KCP-Edge) controllers.
 
 ```console
-$ bin/remove-location.sh -h
-bin/remove-location.sh usage: objname
+$ kubectl kcp-edge remove location -h
+Usage: kubectl kcp-edge remove location ($kubectl_flag | --imw ws_path)* objname
 
 $ kubectl ws root:imw-1
 Current workspace is "root:imw-1".
 
-$ bin/remove-location.sh demo1
+$ kubectl kcp-edge remove location demo1
 synctarget.workload.kcp.io "demo1" deleted
 location.scheduling.kcp.io "demo1" deleted
 
-$ bin/remove-location.sh demo1
+$ kubectl kcp-edge remove location demo1
```

Comment:
Is this line intentional (to show the script won't do anything unnecessary)?

Reply:
Exactly.

Reply:
Cool.